frontend/apps/cloudstorage/cloudstorage.lua

```diff
@@ -168,6 +157,10 @@ function CloudStorage:openCloudServer(url)
     end
     if tbl and #tbl > 0 then
         self:switchItemTable(url, tbl)
+        self:setTitleBarIconText("home")
```

Comment:
Text? Is that a deceptive function name?

Reply:
The function can set **icon** in the button and **text** in the title.

Reply:
Would read easier if it was named `setTitleBarIconAndText()` :)